Specifications/Language/3_Expressions/CopyAndUpdateExpressions.md

```diff
@@ -12,63 +12,61 @@ They are constructed using a ternary operator `w/` `<-`; the syntax `w/` should
 
-The type of the `modification` expression needs to match the type of the named item that diverges. 
-For instance, if `complex` contains the value `Complex(0., 0.)`, where the type `Complex` is defined [here](https://github.com/microsoft/qsharp-language/blob/main/Specifications/Language/1_ProgramStructure/2_TypeDeclarations.md#type-declarations), then 
+The type of the `modification` expression needs to match the type of the named item that diverges.
+For instance, if `complex` contains the value `Complex(0., 0.)`, where the type `Complex` is defined [here](https://github.com/microsoft/qsharp-language/blob/main/Specifications/Language/1_ProgramStructure/2_TypeDeclarations.md#type-declarations), then
 
 ```qsharp
 complex w/ Re <- 1. 
 ```
 
-is an expression of type `Complex` that evaluates to `Complex(1.,0.)`.
+is an expression of type `Complex` that evaluates to `Complex(1., 0.)`.
 
 ## Copy-and-update of arrays
 
 For arrays, `itemAccess` can be an arbitrary expression of a suitable type;
-the same types that are valid for array slicing are valid in this context. Concretely, the `itemAccess` expression can be of type `Int` or `Range`. If `itemAccess` is a value of type `Int`, then the type of `modification` has to match the item type of the array. If `itemAccess` is a value of type `Range`, then the type of `modification` has to be the same as the array type. 
+the same types that are valid for array slicing are valid in this context. Concretely, the `itemAccess` expression can be of type `Int` or `Range`. If `itemAccess` is a value of type `Int`, then the type of `modification` has to match the item type of the array. If `itemAccess` is a value of type `Range`, then the type of `modification` has to be the same as the array type.
 
-For example, if `arr` contains an array `[0,1,2,3]`, then 
+For example, if `arr` contains an array `[0, 1, 2, 3]`, then
 
-- `arr w/ 0 <- 10` is the array `[10,1,2,3]`.
-- `arr w/ 2 <- 10` is the array `[0,1,10,3]`.
-- `arr w/ 0..2..3 <- [10,12]` is the array `[10,1,12,3]`.
+- `arr w/ 0 <- 10` is the array `[10, 1, 2, 3]`.
+- `arr w/ 2 <- 10` is the array `[0, 1, 10, 3]`.
+- `arr w/ 0..2..3 <- [10, 12]` is the array `[10, 1, 12, 3]`.
 
-Copy-and-update expressions allow the efficient creation of new arrays based on existing ones. 
-The implementation for copy-and-update expressions avoids copying the entire array 
+Copy-and-update expressions allow the efficient creation of new arrays based on existing ones.
```

Comment:
I'm not sure I'd agree with the word "efficient" here, but that's not really in scope right now ;)